Upstream-Status: Backport

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 81febee22e4e0f533b5452e068c9b6d946b9a61f Mon Sep 17 00:00:00 2001
From: Daniel Playfair Cal <daniel.playfair.cal@gmail.com>
Date: Fri, 13 Dec 2019 11:48:23 +0000
Subject: [PATCH] Add missing algorithm header in
 bitmap_cursor_factory_ozone.cc

This is needed for the use of std::find_if

Change-Id: Ica47d3d323f5b2e115463372824b1af84b2294bc
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1963832
Commit-Queue: Maksim Sisov <msisov@igalia.com>
Reviewed-by: Maksim Sisov <msisov@igalia.com>
Reviewed-by: Robert Kroeger <rjkroege@chromium.org>
Auto-Submit: Daniel Playfair Cal <daniel.playfair.cal@gmail.com>
Cr-Commit-Position: refs/heads/master@{#724589}
---
 AUTHORS                                             | 1 +
 ui/base/cursor/ozone/bitmap_cursor_factory_ozone.cc | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/AUTHORS b/AUTHORS
index 2b694a363391..1db51a9a7369 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -205,6 +205,7 @@ Daniel Johnson <danielj41@gmail.com>
 Daniel Lockyer <thisisdaniellockyer@gmail.com>
 Daniel Nishi <dhnishi@gmail.com>
 Daniel Platz <daplatz@googlemail.com>
+Daniel Playfair Cal <daniel.playfair.cal@gmail.com>
 Daniel Shaulov <dshaulov@ptc.com>
 Daniel Trebbien <dtrebbien@gmail.com>
 Daniel Waxweiler <daniel.waxweiler@gmail.com>
diff --git a/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.cc b/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.cc
index 71ec564b4803..f79d7932e236 100644
--- a/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.cc
+++ b/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.cc
@@ -4,6 +4,8 @@
 
 #include "ui/base/cursor/ozone/bitmap_cursor_factory_ozone.h"
 
+#include <algorithm>
+
 #include "base/logging.h"
 #include "third_party/skia/include/core/SkBitmap.h"
 #include "ui/base/cursor/cursors_aura.h"
-- 
2.20.1

